code/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -396,13 +396,13 @@
 /datum/reagent/consumable/ethanol/booger
 	name = "Booger"
 	id = "booger"
-	description = "Ewww..."
+	description = "Eww..."
```

Comment:
Eh? This isn't really 'wrong', it's subject to interpretation how many w's you want... it's used for accentuation.

Reply:
It's not that it's wrong, it's just that having 3 w's causes it to create a hyperlink like in the image I attached. Possibly because the chat window is designed to make anything starting with "www." change into a link.

Reply:
Ahah.